multi-agent/adapters/outbound/gemini/file_upload_adapter.py

```
@@ -0,0 +1,104 @@
+                    file_name=files[i].file_name,
+                    mime_type=files[i].mime_type,
+                    file_uri=results_by_index[i].uri,
+                    size_bytes=len(files[i].file_bytes),
+                )
+                for i in range(len(files))
+            ]
+        except Exception as e:
+            for resp in uploaded:
+                try:
+                    self._client.files.delete(name=resp.name)
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)